templates/example_pixel_cell_clustering.ipynb

```diff
@@ -0,0 +1,679 @@
+{
```

Comment:
<p>For this, I think people will either want an experiment-specific prefix, or the datetime option. </p><p> </p><p>We should change the logic to default to ask for a prefix (like "default_pixel_SOM"), which gets put on the front (not the back) of the directory name. If it's left as None, then all three get set to date time</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/428/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>